client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/protocol/InternalFlag.java

```diff
@@ -0,0 +1,26 @@
+package org.infinispan.client.hotrod.impl.protocol;
+
+/**
+ * Flags which are internal to the protocol and shouldn't be exposed to users.
```

Comment:
org.infinispan.client.hotrod.Flag is visible to API users and I don't think they should/be-able-to use NANO_DURATIONS directly. This is why I've added InternalFlags. The codec will set the flag if one of the durations is not multiple of second otherwise seconds will be used.